src/bonsai/websocket/markets.ts

```
@@ -122,10 +169,14 @@ export function setUpMarkets(store: RootStore) {
   return createStoreEffect(store, selectWebsocketUrl, (wsUrl) => {
     const unsub = subscribeToWsValue(MarketsValueManager, { wsUrl }, (val) => {
       const hasData = val.data != null;
+      // Only set markets if we haven't already set them via REST call
+      // HACK: disable websockets temproarily
       if (hasData && !lastSetHadData) {
         setMarkets(val);
-      } else {
-        throttledSetMarkets(val);
+      } else if (hasData && lastSetHadData) {
+        // Don't overwrite the transformed data with raw websocket data
+        console.log("Websocket received market updates, but keeping transformed data from REST call");
```

ESLint

[error] Unexpected console statement.

(no-console)